playbook/app/pb_kits/playbook/pb_date_picker/date_picker_helper.ts

```diff
@@ -164,17 +175,40 @@ const datePickerHelper = (config: DatePickerConfig, scrollContainer: string | HT
     onYearChange: [() => {
       yearChangeHook()
     }],
-    plugins: setPlugins(),
+    plugins: setPlugins(thisRangesEndToday),
     position,
     positionElement: getPositionElement(positionElement),
     prevArrow: '<i class="far fa-angle-left"></i>',
     static: staticPosition,
   })
 
+
   // ===========================================================
   //                 Additional JS Functionality               |
   // ===========================================================
 
+  // opens flatpickr instance when calander icon is clicked so we can have a hover state on icon
```

Comment:
Looks great! If this code isn't needed, can we just remove it instead of leaving it commented out? @carloslimasd